### PR TITLE
Add health-check for external balancers

### LINKED ISSUE #172: Add basic http healthcheck for external balancers
Most external balancers can perfrom http healthchecks and parse non-200 response statuses. I suggest to expose basic healthcheck endpoint in metrics cartridge role. I'd suggest the following:

- [ ] add 503/200 http handler based on snippet below
- [ ] allow to enable/configure url in clusterwide config
- [ ] don't forget that this endpoint should not require cartridge authentication
- [ ] add it to cluster-cli default template
- [ ] add to documentation basic nginx/haproxy configurations to utilize it 

Snippet:
```
local httpd = cartridge.service_get('httpd')
    httpd:route({method = 'GET', path = '/health'}, function()
        local member = require('membership').myself()
        if box.info.status and box.info.status == 'running' and member and
            member.status and member.status == 'alive' and member.payload and
            member.payload.state and member.payload.state == 'RolesConfigured' then
            return {body = 'OK', status = 200}
        else
            return {body = 'Router is dead', status = 500}
        end
    end)
```

### PR DESCRIPTION
**Pull request by no1seman**
In production environments Tarantool Cluster usually has a large number of so called "routers", Tarantool instances that handle input load and it is required to evenly distribute the load. Various load-balancers are used for this, but any load-balancer have to know which "routers" are ready to accept the load at that very moment. Metrics library has a special plugin that creates an http handler that can be used by the load-balancer to check the current state of any Tarantula instance. If the instance is ready to accept the load, it will return a response with a 200 status code, if not, with a 500 status code.

I didn't forget about

- [X] Tests
- [X] Changelog
- [X] Documentation

Close #172

### CODE REVIEW

File: metrics-scm-1.rockspec [hunk 1]
```diff
@@ -35,6 +35,7 @@ build = {
         ['metrics.plugins.graphite']                     = 'metrics/plugins/graphite/init.lua',
         ['metrics.plugins.prometheus']                   = 'metrics/plugins/prometheus/init.lua',
         ['metrics.plugins.json']                         = 'metrics/plugins/json/init.lua',
+        ['metrics.plugins.health']                       = 'metrics/plugins/health/init.lua',
```
Comment (ZolotarevIgor-mail-ru) [inline metrics-scm-1.rockspec:38]: Переименуй  файл init.lua в health.lua

File: test/integration/cartridge_health_test.lua [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    t.skip_if(cartridge_version == 'unknown', 'Cartridge version is unknown, must be v2.0.2 or greater')
+    t.skip_if(utils.is_version_less(cartridge_version, '2.0.2'), 'Cartridge version is must be v2.0.2 or greater')
+end
+
+g.test_cartridge_health_handler = function()
+    check_cartridge_version()
+    upload_config()
+    local main_server = g.cluster:server('main')
+    local resp = main_server:http_request('get', '/health')
+    t.assert_equals(resp.status, 200)
```
Comment (ZolotarevIgor-mail-ru) [inline test/integration/cartridge_health_test.lua:58]: Не хватает теста на 500 статус

File: README.md [hunk 3]
```diff
@@ -129,6 +129,10 @@ local latency = math.random(1, 10)
 http_requests_latency:observe(latency)
 ```
 
+### Instance health check
```
Comment (ZolotarevIgor-mail-ru) [inline README.md:132]: Продублируй это в doc/monitoring/getting_started.rst, чтобы на сайте тоже было

**Review by ZolotarevIgor-mail-ru — CHANGES_REQUESTED** [review]

File: metrics-scm-1.rockspec (hunk 1 above)
Comment (no1seman) [reply metrics-scm-1.rockspec]: plugin file renamed to health.lua

File: test/integration/cartridge_health_test.lua (hunk 2 above)
Comment (no1seman) [reply test/integration/cartridge_health_test.lua:58]: Added test for 500

File: README.md (hunk 3 above)
Comment (no1seman) [reply README.md:132]: Add desc for health handler to doc/monitoring/getting_started.rst

**Review by ZolotarevIgor-mail-ru — APPROVED** [review]

**Pull request merged**